synapse/api/auth/msc3861_delegated.py

```diff
@@ -59,7 +59,7 @@ def scope_to_list(scope: str) -> List[str]:
     return scope.strip().split(" ")
 
 
-class PrivateKeyJWTWithKid(PrivateKeyJWT):
+class PrivateKeyJWTWithKid(PrivateKeyJWT):  # type: ignore[misc]
```

Comment:
We'd need stubs/annotations for authlib for this to go away, I think? Ditto for LogoutToken.

Reply:
Yes, but I figured since there was only 2 instances it was better to enable the option to ignore it twice?